custom-meta-boxes.php

```diff
@@ -22,6 +22,8 @@
 Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+define( 'CMB_VERSION', '1.0.3' );
```

Comment:
Might as well add a check above if you don't mind. Most likely that someone doesn't have this set, but could be possible and could also conflict with the __other__ CMB library